settings/src/components/email-address.js

```diff
@@ -0,0 +1,94 @@
+
+/**
+ * WordPress dependencies
+ */
+import { Button, TextControl, Notice, Spinner } from '@wordpress/components';
+import { useState, useEffect } from '@wordpress/element';
```

Comment:
```suggestion
import { useState } from '@wordpress/element';
```

I don't see `useEffect` being used anywhere.